k-distribution/src/main/scripts/lib/kit/__main__.py

```diff
@@ -0,0 +1,115 @@
+    command_subparsers = argument_parser.add_subparsers(dest='command', required=True)
+
+    init_subparser = command_subparsers.add_parser('init', help='Initialize prover.')
+    init_subparser.add_argument('kompiled_dir', type=dir_path, help='Path to *-kompiled directory.')
+    init_subparser.add_argument('main_file', type=file_path, help='Path to main definition file.')
+    init_subparser.add_argument('spec_file', type=file_path, help='Path to specification JSON file.')
+    init_subparser.add_argument('-I', '--include', metavar='INCLUDE', dest='include_dirs', type=dir_path, default=[], action='append', help='Path to directory to include.')
+    init_subparser.add_argument('-f', '--force', dest='force', default=False, action='store_true', help='Overwrite existing configuration.')
+
+    expand_subparser = command_subparsers.add_parser('expand', help='Expand node.')
+    expand_subparser.add_argument('node_id', type=str, help='ID of node to expland.')
+
+    # Dummy command for testing
+    command_subparsers.add_parser('prove', help='Load prover from config file.')
+
+    return argument_parser
+
+
+def init(args):
+    if not args.work_dir.exists():
+        raise ValueError(f'Working directory doe not exist: {args.work_dir}')
+
+    work_dir = args.work_dir / '.kit'
+    if not args.force and work_dir.exists():
+        raise ValueError(f'Summary already exists at: {work_dir}')
+
+    # Init CFG
+    cfg_file = work_dir / 'cfg.json'
+
+    claim = _extract_claim_from_spec_file(args.spec_file)
+    cfg = _create_cfg_from_claim(claim)
+
+    # Create config
+    config_file = work_dir / 'config.json'
+
+    config = {
+        'kompiledDir': relpath(args.kompiled_dir, work_dir),
+        'mainFile': relpath(args.main_file, work_dir),
+        'cfgFile': relpath(cfg_file, work_dir),
+        'includeDirs': [relpath(include_dir, work_dir) for include_dir in args.include_dirs],
+    }
+
+    work_dir.mkdir(exist_ok=True)
+
+    with open(config_file, 'w') as f:
+        f.write(json.dumps(config, indent=4))
+        f.write('\n')
+
+    with open(cfg_file, 'w') as f:
+        f.write(cfg.to_json())
+        f.write('\n')
+
+
+def _extract_claim_from_spec_file(spec_file: Path) -> KClaim:
+    with open(spec_file, 'r') as f:
+        spec = json.load(f)
+
+    module = KFlatModule.from_dict(spec['term'])
+    claims = [sentence for sentence in module if type(sentence) is KClaim]
+
+    if len(claims) != 1:
+        raise ValueError(f'Expected exactly one claim in {str(spec_file)}, found: {len(claims)}')
+
+    claim = claims[0]
+    claim = claim.let(body=eliminate_generated_top(claim.body), att=EMPTY_ATT)
+    return claim
+
+
+def _create_cfg_from_claim(claim: KClaim) -> KCFG:
+    init, target = ConstrainedTerm.from_claim(claim)
+    cfg = KCFG()
+    _, init_id = cfg.insertNode(init.as_term())
+    _, target_id = cfg.insertNode(target.as_term())
+    cfg.init = init_id
+    cfg.target = target_id
```

Comment:
`target` states needs to be a list or set:

```suggestion
    cfg.target = [target_id]
```